operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1097,6 +1113,22 @@ ruler:
   env: []
   extraEnvFrom: []
 
+  kedaAutoscaling:
+    enabled: false
+    minReplicaCount: 1
+    maxReplicaCount: 10
+    pollingInterval: 10
+    targetCPUUtilizationPercentage: 80
+    targetMemoryUtilizationPercentage: 80
+    customHeaders: {}
+      # X-Scope-OrgID: ""
+    behavior:
+      scaleDown:
+        policies:
+        - periodSeconds: 60
```

Comment:
The value in jsonnet is 600 here I think. The reason is that the ruler is a bit more sensitive to scaling up and down. Restarting a ruler can result in missed rule evaluations.